README.md

```diff
@@ -21,27 +21,27 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+This is a migration file. The numbers at the beginning are a timestamp of when the file was created. Migration files are used for table setup.
 ```
 
 ### Question 3
 
 In a Rails application, how is the router related to controller actions?  
 
 ```text
-Your answer...
+The router creates paths for each controller action.
 ```
 
```

Comment:
👍  When a request comes in, the router checks to see if it has a route that matches the request, if so it maps the request to a specific controller action.